Sources/ProfileRecorderSampleConversion/HelperFunctions.swift

```diff
@@ -20,3 +20,64 @@ import Foundation
 func swift_reportWarning(_ dunno: Int, _ message: String) {
     fputs("WARNING: \(message)\n", stderr)
 }
+
+extension Substring.UTF8View {
+    /// This is a very terrible JSON parser that should just about be able to parse a `StackFrame`
+    ///
+    /// - note: We are ignoring the stack pointer, so the stack pointer field will always be 0.
+    public func attemptFastParseStackFrame() -> StackFrame? {
+        enum State {
+            case beginning
+            case ipWaitingFor0x
+            case ipStartRange(String.Index, String.Index?)
+
+            var isWaitingFor0x: Bool {
+                if case .ipWaitingFor0x = self {
+                    return true
+                }
+                return false
+            }
+        }
+        var state: State = .beginning
+        let bytes = self
+        loop: for byteIndex in bytes.indices {
+            let byte = bytes[byteIndex]
+            switch byte {
+            case UInt8(ascii: "\""):
+                if case .ipStartRange(let start, .none) = state {
+                    state = .ipStartRange(start, byteIndex)
+                    break loop
+                }
+                continue
+            case UInt8(ascii: "\""), UInt8(ascii: " "), UInt8(ascii: ":"), UInt8(ascii: "{"), UInt8(ascii: "}"),
```

Comment:
```suggestion
            case UInt8(ascii: " "), UInt8(ascii: ":"), UInt8(ascii: "{"), UInt8(ascii: "}"),
```

`UInt8(ascii: "\"")` already has a `case` on L46